spec/features/editing_content/insert_contact_no_js_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.feature "Insert contact without Javascript" do
+  include AccessibleAutocompleteHelper
+
+  scenario do
+    given_there_is_an_edition
+    when_i_go_to_edit_the_edition
+    and_i_go_to_add_a_contact
+    when_i_select_a_contact
+    then_i_see_the_contact_markdown_snippet
+  end
+
+  let(:organisation) { { "content_id" => SecureRandom.uuid, "internal_name" => "Organisation" } }
```

Comment:
Worth mentioning that we don't use lets or befores in our feature specs. Things like this usually get shoved in one of the early steps, before they are needed. If we need to refer to them later, we use instance '@' variables.

Reply:
Hmm.. again these are the sort of things I carried this from our previous feature spec. Happy to update them, but I will need help.

Reply:
Refactored as discussed. Do let me know if they need any updates.